File: src/test/java/io/plagov/advent/aoc2025/Day01Test.java
package io.plagov.advent.aoc2025;

import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;
import util.FileUtil;

import java.util.List;

public class Day01Test {

    private final List<String> input = FileUtil.readInputFile("2025/day01.txt");

    @Test
    void solveFirst() {
        var day1 = new Day01();
        Assertions.assertEquals(1034, day1.partOne(input));
    }
}

File: src/test/kotlin/io/plagov/advent/aoc2024/Day01Test.kt
package io.plagov.advent.aoc2024

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test
import util.readInputFile

class Day01Test {

  private val input = readInputFile("2024/day01.txt")

  private val day1 = Day01()

  @Test
  fun solveFirst() {
    Assertions.assertEquals(1879048, day1.partOne(input))
  }

  @Test
  fun solveSecond() {
    Assertions.assertEquals(21024792, day1.partTwo(input))
  }
}
